modules/ROOT/pages/add-api-alert-task.adoc

```diff
@@ -1,21 +1,27 @@
-= Adding an API Alert
+= Add API Alerts
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-. In API Manager 2.x, in API Administration, select an API instance, and on the API instance dashboard, select Alerts.
-. Click Add Alert and set the alert options:
-* Name: Type an alert name, for example `red-alert`.
-* Enabled: Accept the default, Enabled, for example.
-* Severity: Accept the default, Critical.
-* Alert type: Select Request Count, for example.
-* When number of occurrences are: Select Greater.
-* than: Select 1.
-* For at least: Select an entry in the range 1 - 999999999 to specify the time intervals as a sliding windows.
-* Recipients: Select the names of users in your business group, or type the email addresses, of users who will receive notification when the alert fires. Press RETURN.
+You can add one or more API alerts for your API instances to track and manage APIs.
+
+To add an API alert:
+
+. Navigate to *API Manager* > *API Administration* and click the API instance for which you want to add an alert.
+. From the left navigation select *Alerts* > *Add alert*.
+. Specify the alert options:
+* *Name*: Type an alert name, for example `trial-alert`.
+* *Enabled*: Accept the default option: *Enabled*.
 +
-. Create the alert.
+If you no longer need the alert, you can disable it any time later.
+* *Severity*: Select a severity for the alert, for example, *Info*.
+* *Alert type*: Select the type of alert to create, for example, *Request Count*:
```

Comment:
```suggestion
* *Alert type*
+
Select the type of alert to create, for example, *Request Count*.
```
What follows does not make sense, because this step tells the audience to select a type of alert to create, names an example, but then goes on to detail THAT example, which is not going to help anyone who chooses a different type of alert to create.

This needs to be reworked so that is made clear, that the three sublisted items here apply ONLY to the one example and not all types offered in the UI control.

I suggest that omit these sublist items altogether.

You could also alter this main list item as follows:
```suggestion
* *Alert type*
+
Select the type of alert to create (for example, *Request Count*), and provide appropriate alert configuration values.
```
Your other alternative is to follow the main list item, *Alert type*, with a text that clarifies that the sublist items documented here are only the configuration options for the *Request Count* alert, and that other alerts will have different configuration items.

Reply:
Done! Thanks!